app/services/to_collection_form/mapper.rb

```diff
@@ -26,8 +26,8 @@ def params
         lock: cocina_object.lock,
         title: CocinaSupport.title_for(cocina_object:),
         description: CocinaSupport.abstract_for(cocina_object:), # Cocina abstract maps to Collection description
-        contact_emails_attributes: CocinaSupport.contact_emails_for(cocina_object:),
-        related_links_attributes: CocinaSupport.related_links_for(cocina_object:),
+        contact_emails_attributes: ToForm::ContactEmailsMapper.call(cocina_object:),
+        related_links_attributes: ToForm::RelatedLinksMapper.call(cocina_object:),
```

Comment:
(This is why I asked the question above. We see here a mix of using `CocinaSupport#method` and `ToForm::FooMapper.call`.)

Reply:
Within To*Form classes, you will not see CocinaSupport. Outside To*Form classes, you may see a small amount of CocinaSupport.

We'll see how this works and can adjust if it ends up unclear.

Reply:
The crux of the problem is that we're trying to extract metadata from cocina AND map to form data structures. Either we strictly separate these, which create a LOT of extra code or we accept a little inelegancy when only trying to extract metadata from cocina.